src/DynamoCoreWpf/Utilities/WpfUtilities.cs

```diff
@@ -115,5 +115,17 @@ public static void FindChildren<T>(DependencyObject parent, string childName, Li
                 }
             }
         }
+
+        public static bool IsControlUserVisible(FrameworkElement element, FrameworkElement container, bool ignoreWidth = false)
```

Comment:
A better name would be `IsElementVisible`, also please document the method, by looking at this I am not sure why do we ignore width in some cases.